### PR TITLE
Book create

### PR DESCRIPTION
**Pull request by Daphineatim**
### In this milestone the following were implemented;

-  Created a Book class in a separate .rb file.
- Created a Label class with an association to the Item class (in a separate .rb file).
- All Book class properties are defined and set up in the constructor method.
- All Label class properties are defined and set up in the constructor method.

- [ ] Implemented methods:

- Added_item method in the Label class
- Should take an instance of the Item class as an input
- Should add the input item to the collection of items
- Should add self as a property of the item object (by using the correct setter from the item object)
- Can_be_archived?() in the Book class
- Should override the method from the parent class
- should return true if parent's method returns true OR if cover_state equals to "bad"
otherwise, it should return false

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  def add_a_book
+    puts "\nPublisher\n"
+    publisher = gets.chomp
+    puts "\nCover state\n"
+    cover = gets.chomp
+    puts "\nPublish date\n"
+    date = gets.chomp
+    book = Book.new(publisher, cover, date)
+    @books.push(book)
+    puts "\nWould you like to add a label?\n"
```
Comment (svitalis123) [inline app.rb:36]: @Daphineatim optionally give the client options such as yes or no, this prevents invalid inputs.

File: main.rb [hunk 2]
```diff
@@ -49,6 +49,21 @@ def self.menu
   end
 end
 
+def starting(action)
```
Comment (svitalis123) [inline main.rb]: Incorporate this in the above case iteration. To keep the code DRY
Reference [here](https://www.itexico.com/blog/software-development-kiss-yagni-dry-3-principles-to-simplify-your-life)

**Review by svitalis123 — CHANGES_REQUESTED** [review]
Hi 👋  [@Daphineatim ] Good Job so far
There are some issues that you still need to work on to go to the next project but you are almost there!

# Highlights 👏 
- Good github flow
- Professional PR
- Good code flow

# Required changes ♻️ 

- Please Kindly don't repeat your code in Main.rb To keep The code DRY
- Please Kindly Remove book class in Item to avoid violating Single responsibility principle

Optional suggestions
 1. Good implementation of the interface. make it better by working on giving users options e.g `to proceed press 0 or 1` to avoid invalid data
> Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@svitalis123 ] in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: app.rb (hunk 1 above)
Comment (Daphineatim) [reply app.rb:36]: Thank you for the feedback.

File: main.rb (hunk 2 above)
Comment (Daphineatim) [reply main.rb]: I have worked on it, thank you for the feedback.

**Review by svitalis123 — APPROVED** [review]
Hi [@Daphineatim ]

To highlight 👍🏽
✔️ Your readme is professional

Your project is complete! There is nothing else to say other than... it's time to merge it 🚀
Congratulations! 🎉



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@svitalis123 ] in your question so I can receive the notification.

**Pull request merged**